qiskit/scheduler/utils.py

```diff
@@ -14,113 +14,15 @@
 
 """Scheduling utility functions."""
 
-from typing import Dict, List, Optional, Union
-from qiskit.pulse import Schedule, MemorySlot, Acquire, PulseError, InstructionScheduleMap
-from qiskit.pulse.commands import AcquireInstruction
+# pylint: disable=invalid-name
+from qiskit.util import deprecate_function
+from qiskit.pulse import macros, utils
 
+format_meas_map = deprecate_function(
+    '"format_meas_map" has been moved to "qiskit.pulse.utils"')(utils.format_meas_map)
```

Comment:
I thought pulse should be kept agnostic to the backend. I guess by putting the builder in there, pulse is no longer agnostic to hardware the way it was before -- but why move `format_meas_map`? Isn't it only used for the scheduler?

Reply:
It was only used by the macros, see my comment above.